ext/ffi/lib.rs

```diff
@@ -260,6 +261,55 @@ enum NativeType {
   Pointer,
   Buffer,
   Function,
+  Struct(Vec<NativeType>),
```

Comment:
I'd expect this to be `Strucr(Box<[NativeType]>)`, as it's smaller.
The size difference affects every instance of the enum.